astropy/utils/data_info.py

```diff
@@ -100,9 +100,7 @@ def dtype_info_name(dtype):
     """
     dtype = np.dtype(dtype)
     if dtype.names is not None:
-        return "({})".format(
-            ", ".join(dtype_info_name(dt[0]) for dt in dtype.fields.values())
-        )
+        return f"({', '.join(dtype_info_name(dt[0]) for dt in dtype.fields.values())})"
```

Comment:
Not an improvement - here, the original really seems a lot clearer. I'd perhaps do
```
info_names = ', '.join(dtype_info_name(dt[0]) for dt in dtype.fields.values())
return f"({info_names}"
```

Reply:
Done